README.md

```diff
@@ -41,6 +41,24 @@ The task board encompasses all the ongoing projects of the team and is managed u
 -   __Done__  
     Once an item is considered done according to the Definition of Done it will be moved to the "Done" stage either by closing or merging the item or manually moving it on the board.
 
+## Estimation
+
+The following Fibonacci point scale is used for estimating the work required for completing a backlog item:
+
+Points  | Meaning
+---:    | :---
+__1__   | Can solve the issue quickly without needing to research
+__2__   | Can solve the issue by researching a little
```

Comment:
Yeah it's a relative scale so it might be pointless to define it further before we move it to the sprint backlog. I don't have anything else to add.